test/fail_compilation/fail19181.d

```diff
@@ -0,0 +1,16 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail19181.d(15): Error: undefined identifier LanguageError
```

Comment:
I think the tests suite will probably fail because LanguageError needs to be surrounded with tick marks.  We'll find out soon enough.

Reply:
Since the other tests have that too, I've just gone ahead and added it. Can't hurt.